apps/spruce/src/pages/task/metadata/index.tsx

```diff
@@ -353,7 +356,24 @@ export const Metadata: React.FC<Props> = ({ error, loading, task, taskId }) => {
 
       {!isDisplayTask && (
         <MetadataCard>
-          <MetadataTitle>Host Information</MetadataTitle>
+          <MetadataTitle>Host Information</MetadataTitle>{" "}
```

Comment:
Did you give much thought to the order of these links? My inclination might be to put host ID first since that's probably the most useful/familiar to users. Potentially

1. ID
2. Distro
3. Image

would introduce a sense of hierarchy too.

Reply:
Reordered like this:
<img width="295" alt="Screenshot 2024-09-09 at 11 25 53 AM" src="https://github.com/user-attachments/assets/5b51116f-ae01-4eff-892b-8359ed623f97">

Also unrelated but removed the bolded label on the patch external links because I don't really like it

Before:

<img width="295" alt="Screenshot 2024-09-09 at 11 24 27 AM" src="https://github.com/user-attachments/assets/1505a0e3-2452-4715-9924-48feb02a9755">

After:

<img width="291" alt="Screenshot 2024-09-09 at 11 24 35 AM" src="https://github.com/user-attachments/assets/3ba05970-a587-429a-982c-2ef177e76f61">